### DISCUSSION TITLE
Tags Component bit BlazorUI

### DISCUSSION THREAD

**Discussion by baddri** [bitfoundation/bitplatform#7388] [General]
Hi, I would like to know if there is a component similar to [tags](https://www.w3schools.com/w3css/w3css_tags.asp) because I can’t find any in the documentation. If there isn’t any, are you planning to implement it in the future?

**Comment by msynk** [comment]
Thanks for the suggestion, it's a good idea. we will take a look into it and come back to you soon.

**Reply by baddri** [reply]
Lovely. That would make a great addition to the library. I really appreciate it.

**Comment by msynk** [comment]
@baddri The issue has been filed: https://github.com/bitfoundation/bitplatform/issues/7479
